bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]; then
+    echo "Usage: bootstrap.sh SAMPLE_PROJECT_FOLDER"
+    exit 1
+fi
+
+DEST=$1
+
+if [ -a $DEST ]; then
```

Comment:
`DEST` may contain special chars (ie. spaces)

Please, surround all `$DEST` by double quotes

```suggestion
if [ -a "$DEST" ]; then
```

Reply:
Applied.